wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
+The reader, however, can still read the text by highlighting over the text.
+
+- Dedicated button: ![Spoiler button](img/spoiler.jpg "Spoiler button")
+- Notes:
+  - If you use the [colour](#colour) tag along with this, the colour will be used with a black background.
+- Syntax:
+```prolog
+[spoiler] ... [/spoiler]
+```
+
+### Spoilerbox
+
+_Not to be confused with [Box](#box) or [Spoiler](#spoiler)._
+
+This will hide a paragraph from the reader's view.
+The reader, however, can still read the paragraph by clicking on the spoilerbox to toggle the hidden paragraph.
+
+- Dedicated button: ![Spoilerbox button](img/spoilerbox.jpg "Spoilerbox button")
+- Notes:
+  - This will create a [Box](#box) with the heading text of `collapsed text`.
+- Syntax:
+```prolog
+[spoilerbox]
+...
+[/spoilerbox]
+```
+
+### Box
+
+_Not to be confused with [Spoilerbox](#spoilerbox)._
+
+This will hide a paragraph from the reader's view.
+The reader, however, can still read the paragraph by clicking on the box to toggle the hidden paragraph.
+
+These are commonly used to hide large images or a large amount of images.
+
+- Dedicated button: ![Box button](img/box.jpg "Box button")
+- Notes:
+  - By default, this will create a box with the no heading text.
+    - If you do leave the `NAME` argument blank (or do not include one), the height of the box will be very small!
+  - The `NAME` argument does not need quotation marks.
+  - The `NAME` argument can contain spaces.
+- Syntax:
+  - Where `NAME` is a name for the box.
+```prolog
+[box=NAME]
+...
+[/box]
+```
+
+### Quote
+
+This will add a quote.
+
+- Dedicated button: ![Quote button](img/quote.jpg "Quote button")
+- Notes:
+  - It is nice to note the author of the quote if you are quoting someone (use `NAME` argument).
+  - The `NAME` argument **must** use quotation marks.
+  - There is a quote button per post (bottom-right corner) that will automatically quote the text of a user.
+    - If you did this, the post (when you click `Post`) will be posted in the current thread.
+- Syntax:
+  - Where `NAME` is a name for the box (**must** be wrapped in quotation marks).
+```prolog
+[quote=NAME]
+...
+[/quote]
+```
+
+### Code
+
+This will add format text with a monospaced font-family and place it inside a gray box.
+This is useful when you are posting code for a storyboard or source code from a project.
+
+- Dedicated button: ![Code button](img/code.jpg "Code button")
+- Notes:
+  - This will preserve white spacing, this means that none of the lines will break unless you break it.
+  - Overflowing text lines will enable a horizontal scroll box.
+- Syntax:
+```prolog
+[code]
+...
+[/code]
+```
+
+### Centre
+
+This will centre a paragraph.
+This is best used for things such as the title or a heading.
+
+- Dedicated button: ![Centre button](img/centre.jpg "Centre button")
+- Notes:
+  - The tags **must** be spelt as `centre` not `center` (note the last two letters).
```

Comment:
I would, but **bold** on the osu-wiki doesn't really stand out if they are part of a word that isn't bolded entirely.